folly/Traits.h

```diff
@@ -384,10 +384,24 @@ struct Bools {
   }
 };
 
+template <bool... Bs>
+struct AllFalse : std::is_same<Bools<Bs...>, Bools<(Bs && false)...>> {};
```

Comment:
I think none. Compiler claims that the parameter pack is not expanded if those are aliases